### ISSUE TITLE
`pure_getters: true` minifies `import.meta` to `import`

### ISSUE THREAD

**Issue by sapphi-red** [terser/terser#1349]
**Bug report or Feature request?**
bug

**Version (complete output of `terser -V` or specific git commit)**
5.16.4

**Complete CLI command or `minify()` options used**
```js
const options = {
  compress: { pure_getters: true },
  module: true,
}
```

[stackblitz reproduction](https://stackblitz.com/edit/node-n7mmjd?file=package.json,index.js)

**`terser` input**
```js
import.meta;
```

**`terser` output or error**
```js
import;
```
This is not a valid JS code.

**Expected result**
```js
import.meta;
```
or
```js
```
(empty code)

**Comment by fabiosantoscode** [comment]
Interesting! I think `import.meta` is being parsed and treated as a regular object property access.

**Comment by EverStarck** [comment]
uh, what happened here? 🤔 

This is breaking [some code](https://github.com/vitejs/vite/issues/11948)
I'm not sure if anything can be done from @vitejs/plugin-legacy? @sapphi-red

**Comment by ehoogeveen-medweb** [comment]
Issues on GitHub are closed when a commit or pull request that references them is merged. This was fixed right after the release of v5.16.5, so the fix should be available in the next version.

**Comment by sapphi-red** [comment]
@EverStarck I don't think your problem is same with the OP. Please create an issue with reproduction.
